src/background.js

```diff
@@ -208,8 +208,8 @@ Badger.prototype = {
                                   function(subdomain){
                                     return subdomain.endsWith(domain);
                                   });
-        _.each(rmvdSubdomains, function(domain){
-          self.storage.setupHeuristicAction(domain, constants.BLOCK);
+        _.each(rmvdSubdomains, function(subDomain){
```

Comment:
Oh yes, pretty sure it can; the bug was fixed in 43c1c63aee9fb73a2d69518c99b1f2d3ba9668be.